spec/Index.test.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Index from '../src/Index';
+import Index from '../react-client/src/Index';
```

Comment:
@nicola24 I think this change is the cause of the test failing. Both the test and original code are in `/react-client/`. I believe your original import was correct. Try to sere if that will fix the issue.

E.g. This should be your current file structure. so you should only need to use `../src/Index` to get the file.
```
__react-client
|___src
|     |___ index.js
|___spec
      |___ index.test.js      
```